packages/dashboard/src/app/views/myStories/content/stories/index.js

```diff
@@ -89,136 +47,152 @@ const longerListOfStories = forceUniqueIds(
+        'requestNextPage',
+        'duplicateStory',
+        'updateStory',
+        'trashStory',
+        'enablePostLocking',
+        'isGrid',
+        'isThumbnailMode',
+        'isLoading',
+        'Story sort options',
+        'direction',
+        'allPagesFetched',
+        'statusValue',
+        'canViewDefaultTemplates',
+      ],
+    },
+  },
 };
 
 const StorybookLayoutContainer = styled.div`
   margin-top: 40px;
   height: 100vh;
 `;
-export const _default = () => {
+export const _default = (args) => {
   const { pageSize } = usePagePreviewSize({
-    isGrid: true,
+    isGrid: args.isGrid,
+    thumbnailMode: args.thumbnailMode,
   });
+  const filter = {
+    status: args.filterStatus,
+    value: args.statusValue,
+    set: args.setFilter,
+  };
+  const sort = {
+    value: args.sortValue,
+    direction: args.direction,
+    set: args.setSort,
+    setDirection: args.setDirection,
+  };
+  const search = {
+    keyword: args.keyword,
+    setKeyword: args.setKeyword,
+  };
+  const view = {
+    style: args.style,
+    toggleStyle: args.toggleStyle,
+    pageSize,
+  };
+  const page = {
+    value: 1,
+    set: args.setPage,
+    requestNextPage: args.requestNextPage,
+  };
+  const storyActions = {
+    duplicateStory: args.duplicateStory,
+    trashStory: args.trashStory,
+    updateStory: args.updateStory,
+  };
 
+  const defaultProps = {
+    allPagesFetched: args.allPagesFetched,
+    filter: filter,
+    isLoading: args.isLoading,
+    page: page,
+    search: search,
+    sort: sort,
+    stories: args.stories,
+    storyActions: storyActions,
+    view: view,
+  };
   return (
-    <SnackbarProvider>
-      <Layout.Provider>
-        <StorybookLayoutContainer>
-          <Content {...defaultProps} view={{ ...view, pageSize }} />
-        </StorybookLayoutContainer>
-      </Layout.Provider>
-    </SnackbarProvider>
+    <FlagsProvider features={{ ...args.enablePostLocking }}>
+      <SnackbarProvider>
+        <Layout.Provider>
+          <StorybookLayoutContainer>
+            <Content {...args} {...defaultProps} />
+          </StorybookLayoutContainer>
+        </Layout.Provider>
+      </SnackbarProvider>
+    </FlagsProvider>
   );
 };
 
-export const NoStories = () => (
-  <Layout.Provider>
-    <SnackbarProvider>
-      <StorybookLayoutContainer>
-        <Content {...defaultProps} stories={[]} />
-      </StorybookLayoutContainer>
-    </SnackbarProvider>
-  </Layout.Provider>
-);
-
-export const AllDataFetched = () => {
-  const { pageSize } = usePagePreviewSize({
-    isGrid: true,
-  });
-  return (
-    <SnackbarProvider>
-      <Layout.Provider>
-        <StorybookLayoutContainer>
-          <Content
-            {...defaultProps}
-            allPagesFetched
-            view={{ ...view, pageSize }}
-          />
-        </StorybookLayoutContainer>
-      </Layout.Provider>
-    </SnackbarProvider>
-  );
+export const NoStories = _default.bind({});
+NoStories.args = {
+  stories: [],
```

Comment:
I am perfectly happy to make these extra stories for these specific details, but I feel like the idea behind `controls` is to allow the user to switch between these kind of option in order to get the desired view without having to have another story to look at. Do y'all think we should scrap these additional stories and just use `control` to show the various uis?

Reply:
i think scrapping the additional stories to use controls is a great plan!